src/components/Text.tsx

```diff
@@ -286,14 +286,19 @@ const tinyText = (color: 'base' | 'primary' | 'error', primary?: boolean) =>
         ? 'text-primary-semi-dimmed'
         : 'text-formal-accent'
     ),
-    fontFamily(primary ? 'font-primary' : undefined),
+    fontFamily(fontPrimary ? 'font-primary' : undefined),
     fontSize('text-xs'),
     lineHeight('leading-3')
   )
 export function TinyText({
   color,
-  primary,
+  fontPrimary,
   children,
-}: { color?: 'base' | 'primary' | 'error'; primary?: boolean } & ChildrenProp) {
-  return <div className={tinyText(color || 'base', primary)}>{children}</div>
+}: {
+  color?: 'base' | 'primary' | 'error'
+  fontPrimary?: boolean
+} & ChildrenProp) {
+  return (
+    <div className={tinyText(color || 'base', fontPrimary)}>{children}</div>
```

Comment:
Why don't we do `color = 'base'` in the props?

Reply:
You are right. Changed it